core/modules/layout/layout.module

```diff
@@ -2074,193 +2074,193 @@ function layout_flexible_row_styles() {
       'split' => '100:0',
       'bootstrap' => '12:0',
       'region_count' => 1,
-      'name' => 'One full-width region',
+      'name' => t('One full-width region'),
     ),
     'region_10_2' => array(
       'split' => '83:17',
       'bootstrap' => '10:2',
       'region_count' => 2,
-      'name' => 'Two regions 83% : 17%',
+      'name' => t('Two regions 83% : 17%'),
```

Comment:
With each one of these strings being identical except for the non-translatable percentages portion, could we just translate the phrase "Two regions", "Three regions", and "Four regions" and then append the percentages afterwards? e.g.
```
t('Two regions') . ' 83% : 17%',
```

Reply:
Good idea! Thank You!
I commited.